.travis.yml

```diff
@@ -0,0 +1,99 @@
+language: php
+
+php:
+  - 5.5
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+
+env:
+  - TYPO3_VERSION=^6.2 COVERAGE=0
+  - TYPO3_VERSION=^7.6 COVERAGE=0
+  - TYPO3_VERSION=^8.7 COVERAGE=0
+  - TYPO3_VERSION=dev-master COVERAGE=0
+
+matrix:
+  exclude:
+    - env: TYPO3_VERSION=^6.2 COVERAGE=0
+      php: 7.2
+    - env: TYPO3_VERSION=^6.2 COVERAGE=0
+      php: 7.1
+    - env: TYPO3_VERSION=^6.2 COVERAGE=0
+      php: 7.0
+    - env: TYPO3_VERSION=^8.7 COVERAGE=0
+      php: 5.5
+    - env: TYPO3_VERSION=^8.7 COVERAGE=0
+      php: 5.6
+    - env: TYPO3_VERSION=dev-master COVERAGE=0
+      php: 5.5
+    - env: TYPO3_VERSION=dev-master COVERAGE=0
+      php: 5.6
+    - env: TYPO3_VERSION=dev-master COVERAGE=0
+      php: 7.0
+    - env: TYPO3_VERSION=dev-master COVERAGE=0
+      php: 7.1
+  include:
+    - env: TYPO3_VERSION=^7.6 COVERAGE=1
+      php: 5.6
+  allow_failures:
+    - env: TYPO3_VERSION=^6.2 COVERAGE=0
```

Comment:
As long as 6.2 is working please keep them in the normal matrix and don't allow failures otherwise we will not notice when we break support.